detectors/src/main/java/org/jlab/clas/timeline/run_detectors.groovy

```diff
@@ -123,7 +123,8 @@ def engines = [
        new rich.rich_nkp_m(), 
        new rich.rich_npro_m(), 
        new rich.rich_npbar_m()],
-  out_HELICITY: [new helicity.helicity()]
+  out_HELICITY: [new helicity.helicity()],
+  out_TRIGGER: [new trigger.trigger()]
```

Comment:
In fact, the CI isn't even checking this new timeline, since it tests on RG-B data, the only run group to have a separate `run_detectors*.groovy` script.

Reply:
I'll combine these in a separate PR